collectives/models/role.py

```diff
@@ -75,6 +75,17 @@ def all_event_creator_roles(cls):
         """
         return cls.all_activity_leader_roles() + cls.all_moderator_roles()
 
+    @classmethod
+    def get(cls, required_id):
+        """
+        :return: Get a :py:class:`RodeIds` from its id
+        :rtype: :py:class:`RodeIds`
+        """
+        for role_id in cls.all_event_creator_roles():
```

Comment:
all_event_creator_roles() retourne tous les roles actuellement, mais ca ne sera pas forcément le cas dans le futur (par exemple si on ajoute un role secrétaire). Il vaudrait mieux itérer sur toutes les valeurs de l'enum

Reply:
Ok, je vais trouver comment faire.